cmake/ctest/drivers/atdm/van1-tx2/local-driver.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -l
```

Comment:
@e10harvey, I just realized this file is not getting used.  We should likely remove it to avoid confusion.

Reply:
I don't think this file should be removed. local-driver.sh is used in two places: https://github.com/trilinos/Trilinos/pull/7151/files#diff-81f96a09d6a0988a70a0432048a23567R9 and https://github.com/trilinos/Trilinos/pull/7151/files#diff-602f89090dbaf9b43a16cf4d01a61b35R9.

Reply:
@e10harvey, no, you are correct.  I added this file :-)  I was thinking I was looking in the directory cmake/std/atdm/van1-tx2/.